.github/workflows/pr_scan.yml

```diff
@@ -5,27 +5,30 @@ on:
     types: [opened, synchronize, reopened]
 
 jobs:
-  
+
   build:
     env:
       sonarToken: ${{ secrets.SONAR_TOKEN }}
     runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
-    
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: latest-stable
```

Comment:
let's not use any 3rd party code.
Just set `echo '::set-env name=XCODE_VERSION::/Applications/Xcode_12.1.app/Contents/Developer'`